framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/model/components/CommonConfig.java

```diff
@@ -64,6 +66,10 @@ public String getDbPassword() {
         return dbPassword;
     }
 
+    public Boolean getDisableSsl() {
```

Comment:
shouldn't it be `isDisableSsl` ?

Reply:
No, it's part of KieApp model and here are required getter and setters for configuration.